src/main/webapp/app/entities/submission.model.ts

```diff
@@ -62,36 +68,26 @@ export function getLatestSubmissionResult(submission: Submission | undefined): R
  * @param submission
  * @param result
  *
- * @return the index of the latestSubmissionResult or undefined if any of the params was undefined
+ * @return
```

Comment:
I think we can remove this, as the function returns nothing

Reply:
yea, I fixed the doc